src/services/fileStorage/proxy-service.js

```diff
@@ -45,50 +40,47 @@ const {
 	SECURITY_CHECK_SERVICE_PATH,
 } = require('../../../config/globals');
 
-const FILE_SECURITY_CHECK_CALLBACK_URI = url.resolve(
-	API_HOST,	SECURITY_CHECK_SERVICE_PATH,
-);
+const FILE_SECURITY_CHECK_CALLBACK_URI = url.resolve(API_HOST, SECURITY_CHECK_SERVICE_PATH);
 
 const sanitizeObj = (obj) => {
 	Object.keys(obj).forEach((key) => obj[key] === undefined && delete obj[key]);
 	return obj;
 };
 
-const prepareThumbnailGeneration = (file, strategy, userId, { name: dataName },
-	{ storageFileName, name: propName }) => (
+const prepareThumbnailGeneration = (file, strategy, userId, { name: dataName }, { storageFileName, name: propName }) =>
 	ENABLE_THUMBNAIL_GENERATION
 		? Promise.all([
-			strategy.getSignedUrl({
-				userId,
-				flatFileName: storageFileName,
-				localFileName: storageFileName,
-				download: true,
-				Expires: 3600 * 24,
-			}),
-			strategy.generateSignedUrl({
-				userId,
-				flatFileName: storageFileName.replace(
-					/(\..+)$/,
-					'-thumbnail.png',
-				),
-				fileType: returnFileType(dataName || propName), // data.type
-			}),
-		]).then(([downloadUrl, signedS3Url]) => rp.post({
-			url: FILE_PREVIEW_SERVICE_URI,
-			body: {
-				downloadUrl,
-				signedS3Url,
-				callbackUrl: url.resolve(FILE_PREVIEW_CALLBACK_URI, file.thumbnailRequestToken),
-				options: {
-					width: 120,
-				},
-			},
-			json: true,
-		}).catch((err) => {
-			logger.warning(new Error('Can not create tumbnail', err)); // todo err message is lost and throw error
-		}))
-		: Promise.resolve()
-);
+				strategy.getSignedUrl({
+					userId,
+					flatFileName: storageFileName,
+					localFileName: storageFileName,
+					download: true,
+					Expires: 3600 * 24,
+				}),
+				strategy.generateSignedUrl({
+					userId,
+					flatFileName: storageFileName.replace(/(\..+)$/, '-thumbnail.png'),
+					fileType: returnFileType(dataName || propName), // data.type
+				}),
+		  ]).then(([downloadUrl, signedS3Url]) =>
+				rp
+					.post({
+						url: FILE_PREVIEW_SERVICE_URI,
+						body: {
+							downloadUrl,
+							signedS3Url,
+							callbackUrl: url.resolve(FILE_PREVIEW_CALLBACK_URI, file.thumbnailRequestToken),
```

Comment:
Codacy found an issue: [Expected indentation of 6 tabs but found 7.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)